### PR TITLE
JAM-5 : Added isNewRecord() method in jModel

### PR DESCRIPTION
**Pull request by iamtheazizul**

### CODE REVIEW

**Comment by shahriar-prepmock** [comment]
@iamtheazizul please change the PR title into this convention. `Ticket Number : Ticket Title.` . For this PR it should be `JAM-5 : Add newRecord() method in jModel`

And we always try to give a meaningful commit message. Here none can understand anything with the message `Added`.

File: jmodel.php [hunk 1]
```diff
@@ -257,6 +257,19 @@ private function validateColumn($column, $value)
...
+    /**
+     * checks if the record already exists in the table or not
+     * 
+     * if the record exists, it returns false
+     * if a new record is added, it returns true
+     * 
+     * @return bool
+     */
+    public function newRecord()
+    {
```
Comment (dodul) [inline jmodel.php:269]:
```
return $this->table->id() === null;
```

File: jmodel.php [hunk 2]
```diff
@@ -257,6 +257,19 @@ private function validateColumn($column, $value)
...
 
+    /**
+     * checks if the record already exists in the table or not
+     * 
+     * if the record exists, it returns false
+     * if a new record is added, it returns true
+     * 
+     * @return bool
+     */
+    public function newRecord()
```
Comment (dodul) [inline jmodel.php:268]: Please change the method name to `isNewRecord()`

Comment (dodul) [inline jmodel.php:268]: Please add return type

**Review by dodul — CHANGES_REQUESTED** [review]

**Comment by dodul** [comment]
please send PR to https://github.com/Prepmock/jambura-core

**Pull request closed without merging**